### PR TITLE
Dockerized postgres/pgadmin

### PR DESCRIPTION
**Pull request by thekaveman**
Here's a proposal for a simple setup for #9, implemented in `spikes/` since I haven't been part of the repo/project layout discussions. Easy enough to move around.

I opted for two separate Docker containers (one for the PostgreSQL server, one for pgAdmin client) since it makes things a little more reusable/portable.

Nothing special going on here, pretty much straight defaults. But there's plenty of ability to extend - like setting up default databases on container startup, local persistence, etc.

I set this up on Windows 10 with:

```bash
$ docker --version
Docker version 18.06.1-ce, build e68fc7a

$ docker-compose --version
docker-compose version 1.22.0, build f46880fe
```

### CODE REVIEW

**Comment by thekaveman** [comment]
Just updated to the latest Docker for Windows, tested and (still!) works:

```bash
$ docker --version
Docker version 18.09.0, build 4d60db4

$ docker-compose --version
docker-compose version 1.23.2, build 1110ad01
```

File: spikes/docker-postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```bash
+docker-compose up [SERVICE]
+```
+
+Where `[SERVICE]` is `postgres` or `pgadmin`.
+
+To shut everything down:
+
+```bash
+docker-compose down
```
Comment (tienyuan) [inline spikes/docker-postgres/README.md]: I suggest using `docker-compose stop` as the default shutdown command and noting `docker-compose down` will destroy data and any changes made in postgres.

Comment (thekaveman) [reply spikes/docker-postgres/README.md]: Great point @tienyuan, thanks for catching this. Updates made, calling out specifically the destructive nature of `docker-compose down`.

**Comment by thekaveman** [comment]
Another point for consideration:

The environment variables are currently defined directly in the `docker-compose.yml` file.

Compose will automatically source an `.env` file in the same directory as `docker-compose.yml` upon start `up`, as an alternatively option. This makes it possible to e.g. share env variables between services that need the same.

**Pull request merged**